README.md

```diff
@@ -42,6 +43,8 @@ Enjoying Nova? Check out some of our other projects:
 * [Goldilocks](https://github.com/FairwindsOps/Goldilocks) - Right-size your Kubernetes Deployments by compare your memory and CPU settings against actual usage
 * [Pluto](https://github.com/FairwindsOps/Pluto) - Detect Kubernetes resources that have been deprecated or removed in future versions
 * [rbac-manager](https://github.com/FairwindsOps/rbac-manager) - Simplify the management of RBAC in your Kubernetes clusters
+
+Or [check out the full list](https://www.fairwinds.com/open-source-software?utm_source=reckoner&utm_medium=reckoner&utm_campaign=reckoner)
```

Comment:
are `utm_source`, `utm_medium` and `utm_campaign` correct? should that be nova in those fields?

Reply:
Good catch, I'm pinging Robert about the templating repo since this is pulled from there.